eslint.config.mjs

```diff
@@ -0,0 +1,85 @@
+import globals from 'globals'
```

Comment:
Why `.mjs` format?

Reply:
It is just the new commended format that got auto-generated:
https://eslint.org/docs/v8.x/use/configure/configuration-files-new